GitCommands/Git/GitModule.cs

```diff
@@ -1461,29 +1461,29 @@ public string GetPuttyKeyFileForRemote([CanBeNull] string remote)
             return GetSetting($"remote.{remote}.puttykeyfile");
         }
 
-        public ArgumentString FetchCmd([CanBeNull] string remote, [CanBeNull] string remoteBranch, [CanBeNull] string localBranch, bool? fetchTags = false, bool isUnshallow = false, bool prune = false)
+        public ArgumentString FetchCmd([CanBeNull] string remote, [CanBeNull] string remoteBranch, [CanBeNull] string localBranch, bool? fetchTags = false, bool isUnshallow = false, bool pruneRemoteBranches = false, bool pruneRemoteBranchesAndTags = false)
```

Comment:
One last comment:

Isn't that better to change the label to "Prune also tags" (and, so, rename the parameter `pruneAlsoTags`)?

I want to gather the others on that. So don't change anything until we settle on something...

Reply:
An opinion - no strong

--prune-tags should be dependent on --prune in the GUI
The code is handling this correctly, also setting --prune

It is not an error in Git to only have --prune-tags, but it should be:
https://git-scm.com/docs/git-fetch#_pruning

If the 'tags' option is made dependent (shifted to the right, grayed out if brances not is set?), change to _Include tags_

If kept as is, I suggest no change
(This should not be overworked.)